models/Result.js

```diff
@@ -430,7 +430,7 @@ class ResultForAdmin {
     this._secondName = sN;
     this._secondNameNumber = sNN;
     this._spotId = spotId;
-    this._winner = winner;
+    this._winnerId = winnerId;
```

Comment:
Nice. I like storing the raw data and calculating the result at the place it's needed.